### DISCUSSION TITLE
Can this code be used on large picture like 5000*5000pixel?

### DISCUSSION THREAD

**Discussion by zhaogev5** [obss/sahi#670] [Q&A] [answered]
wonderful work and I wonder how can I use this work on some large pic like remote sensing which is huge.

**Comment by kadirnar** [comment]
Which artificial intelligence model do you want to use? Yolov5? Detectron2? MMdet?
Can you send the picture?

**Comment by fcakyon** [comment, marked as answer]
Hello @zhaogev5! Yes `SAHI` can handle large remote sensing pictures, check the paper for results on `xView` satellite imagery dataset: https://arxiv.org/abs/2202.06934

You can follow the notebooks to perform sliced object detection on large images: https://github.com/obss/sahi/tree/main/demo

There is also a blogpost on satellite object detection with `SAHI`: https://blog.ml6.eu/how-to-detect-small-objects-in-very-large-images-70234bab0f98

**Comment by zhaogev5** [comment]
Thank you for your prompt reply！
